backend/src/main/java/com/woowacourse/gongseek/like/presentation/LikeController.java

```diff
@@ -0,0 +1,40 @@
+package com.woowacourse.gongseek.like.presentation;
+
+import com.woowacourse.gongseek.auth.presentation.AuthenticationPrinciple;
+import com.woowacourse.gongseek.auth.presentation.dto.AppMember;
+import com.woowacourse.gongseek.like.application.LikeService;
+import com.woowacourse.gongseek.like.presentation.dto.LikeResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RequestMapping("/api/articles/{articleId}")
+@RestController
+public class LikeController {
+
+    private final LikeService likeService;
+
+    @PostMapping("/like")
+    public ResponseEntity<LikeResponse> likeArticle(
```

Comment:
```suggestion
    public ResponseEntity<LikeResponse> doOnArticle(
```
LikeController과 LikeService에서 likeArticle은 좀 어색하다고 느껴지네요. 주디는 어떤가요??
좋아요 취소도 마찬가지로요!!

Reply:
헉 저는 별로 어색하다고 안느꼈는뎅 팀원들이랑 얘기해봐요!